generatedTypes/index.d.ts

```diff
@@ -35,6 +35,7 @@ export {default as TouchableOpacity, TouchableOpacityProps} from './components/t
 export {default as Button, ButtonPropTypes, ButtonProps} from './components/button';
 export {default as Checkbox, CheckboxPropTypes, CheckboxProps} from './components/checkbox';
 export {default as Chip, ChipPropTypes, ChipProps} from './components/chip';
+export {default as Drawer, DrawerProps, ItemProps} from './components/drawer';
```

Comment:
We can't export a typing called `ItemProps` it's too vague. 
Better rename it to DrawerItemProps

Reply:
Done